frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -230,14 +230,18 @@ string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text
 string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text.diverged-from-and-older-than-remote=(diverged from & older than {0})
 string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text.untracked=(untracked)
 
-string.GitMachete.EnhancedGraphTable.automatic-discover.cant-discover-layout-error-title=Can't automatically discover repository layout
+string.GitMachete.EnhancedGraphTable.automatic-discover.notification.title.cannot-discover-layout-error=Can't automatically discover repository layout
 string.GitMachete.EnhancedGraphTable.automatic-discover.task-title=Automatically discovering branch layout
 string.GitMachete.EnhancedGraphTable.automatic-discover.success-message=Branch layout has been automatically discovered
-string.GitMachete.EnhancedGraphTable.empty-table-text.cant-discover-layout=Provided machete file ({0}) is empty and branch layout can''t be automatically detected
+string.GitMachete.EnhancedGraphTable.empty-table-text.cannot-discover-layout=Provided machete file ({0}) is empty and branch layout can''t be automatically detected
 string.GitMachete.EnhancedGraphTable.empty-table-text.loading=Loading...
 string.GitMachete.EnhancedGraphTable.empty-table-text.only-skipped-in-machete-file=None of the branches provided by machete file ({0}) exists within the local repository
 string.GitMachete.EnhancedGraphTable.skipped-branches-text=The following branches defined by machete file do not belong to the local repository:
 
+string.GitMachete.RediscoverSuggester.task-title="Suggesting rediscover..."
```

Comment:
Hmm it looks like the sole non-commented occurrence of double quotes in the entire file... is this needed? If not, then also consider patching scripts/enforce-property-single-quote-in-bundle-message to forbid them

Reply:
(or add a separate script)